pypeit/images/processrawimage.py

```diff
@@ -38,8 +38,11 @@ class ProcessRawImage(pypeitimage.PypeItImage):
             Dict describing the steps performed on the image
         _bpm (np.ndarray):
             Holds the bad pixel mask once loaded
```

Comment:
See my comment below. The noise model and read noise image should be generated in this function call, not scienceimage.

Reply:
Not this PR